src/popup/router/components/Filters.vue

```diff
@@ -64,17 +77,18 @@ export default {
 @use '../../../styles/mixins';
 
 .filters {
-  height: 50px;
-  width: 100%;
+  height: 36px;
   display: flex;
   gap: 8px;
-  padding: 13px 0;
+  padding-bottom: 12px;
+  background: var(--screen-bg-color);
+  padding-inline: var(--screen-padding-x);
+  margin-left: calc(-1 * var(--screen-padding-x));
+  width: calc(100% + (2 * var(--screen-padding-x)));
```

Comment:
Instead of this I suggest to use margin-right with the same value as margin-left.